iam-login-service/src/main/java/it/infn/mw/iam/api/aup/model/AupRemindersAndSignatureValidator.java

```diff
@@ -50,6 +50,11 @@ public boolean isValid(AupDTO value, ConstraintValidatorContext context) {
       return false;
     }
 
+    if (signatureValidityInDays == 0) {
+      value.setAupRemindersInDays("30,15,1");
```

Comment:
can we move the set out from validation?

Reply:
In alternative, we can both clear reminders value from dashboard PUT request in case validity in days is = 0 and return bad request in case the AupDTO contains validity in days = 0 and a value for reminders.